modules/nw-egress-ips-about.adoc

```diff
@@ -205,7 +207,7 @@ The automatic assignment approach works best for clusters installed in environme
 [id="considerations-manual-egress-ips"]
 == Considerations when using manually assigned egress IP addresses
 
-This approach is recommended for clusters installed in public cloud environments, where there can be limitations on associating additional IP addresses with nodes.
+This approach is recommended for clusters where there can be limitations on associating additional IP addresses with nodes.
```

Comment:
Try to rewrite this to avoid "recommends." It's better to just describe when a user would want to do a thing, not actually recommend it.